modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -556,6 +557,66 @@ The output represents the `name` keys as XML elements.
 
 See also, xref:dataweave-cookbook-output-self-closing-xml-tags.adoc[Example: Outputting Self-closing XML Tags].
 
+[example11]]
+=== Example: Transform XML elements into JSON format and replace characters
+
+This example iterates over an XML file that contains details of employees such as the `Id`, `Name` and `Address`, and you convert the file into JSON format. The DataWeave script uses the `replace` function to replace in the `Address` element the characters `-` and `/` with blank space.
+
+==== Input
+
+The XML input serves as the payload to the DataWeave source. Notice that the `Address` element contains `-` and `/` characters.
+
+[source,xml,linenums]
+----
+<?xml version='1.0' encoding='UTF-8'?>
+<root>
+<employees>
+<Id>1</Id>
+<Name>Mule</Name>
+<Address>MuleSoft Avenue - 123</Address>
+</employees>
+<employees>
+<Id>2</Id>
+<Name>Max</Name>
+<Address>MuleSoft Avenue-456/5/e</Address>
+</employees>
+</root>
+----
+
+==== Source
+
+The DataWeave script iterates over the payload elements, and the `replace` function `/([\-\,\/])/ with " "` replaces the `-` and `/` characters with blank space.
```

Comment:
(Suggestion)
The following DataWeave script iterates over the payload elements and performs a mapping to an object. The instruction 'payload01.Address replace /([\-\,\/])/ with " "' replaces the `-` and `/` characters with blank spaces.